common/org.testeditor.dsl.common/src/org/testeditor/dsl/common/util/classpath/ClasspathUtil.xtend

```diff
@@ -44,22 +44,38 @@ class ClasspathUtil {
 			path = new Path(path.toFile.absolutePath)
 			cpEntry = path.getBuildToolClasspathEntry
 		}
+		if (cpEntry === null) {
+			logger.error("Could not find corresponding classpath entry for path='{}'.", path)
+			return 'com.example'
```

Comment:
Not sure about the `com.example` here. Maybe the default / empty package instead?